scripts/postcss-filter.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+// post-process CSS using PostCSS
+// (https://github.com/postcss/postcss)
+//
+// This adds vendor prefixes using autoprefixer
+// https://github.com/postcss/autoprefixer
+
+require('es6-promise').polyfill();
```

Comment:
Not a blocker for this PR, but we should probably just require Node v0.12 at this point.